build.sh

```diff
@@ -2,8 +2,8 @@
 
 # build the sdk
 mono .paket/paket.exe restore
-msbuild ZFS-SDK.sln /p:Configuration=Release
+msbuild src/ZFS-SDK.sln /p:Configuration=Release
 
 # ensure Z3 has execute access permissions
-chmod +x packages/zen_z3_linux/output/z3
-chmod +x packages/zen_z3_osx/output/z3
+chmod +x packages/zen_z3_linux/output/z3-linux
```

Comment:
added these as they were the only files in the output folders when I compiled (both in osx and linux environment).